cnf-certification-test/networking/suite.go

```diff
@@ -121,8 +121,35 @@ var _ = ginkgo.Describe(common.NetworkingTestKey, func() {
 		testhelper.SkipIfEmptyAny(ginkgo.Skip, env.Pods)
 		testPartnerSpecificTCPPorts(&env)
 	})
+	testID, tags = identifiers.GetGinkgoTestIDAndLabels(identifiers.TestDpdkCPUPinningExecProbe)
+	ginkgo.It(testID, ginkgo.Label(tags...), func() {
+		testhelper.SkipIfEmptyAny(ginkgo.Skip, env.Pods)
```

Comment:
Add a cpuPinningPodsWithDpdk() filter in provider/pods.go to directly skip this test if such pods are not detected since it will be not applicable